code/game/objects/structures.dm

```diff
@@ -165,7 +165,7 @@
 		if(examine_status)
 			. += examine_status
 	if(climbable)
-		. += "<span class='info'>You can <b>Click-Drag</b> someone to [src] to put them on the table after a short delay.</span>"
+		. += "<span class='info'>You can <b>Click-Drag</b> someone to [src] to put them on [src] after a short delay.</span>"
```

Comment:
This should probably use a "it" pronoun, repeating the same thing twice is a bit redundant